ENH: it's useful to log loosers too :)

dohmatob opened the pull request:
Hi @karinqe,

Gr8 code you have there! Please accept this tiny PR. I hope this breaks nothing :)

dohmatob committed:
ENH; it's useful to log loosers too :)

dohmatob committed:
ENH: changed method name to log_winners_and_loosers

karinqe commented:
Hi, thanks for the pull request. Are you actually somehow using the bot? I thought the robopoker competition got discontinued.
As for the patch, could you maybe rename it to something like "log_winners_and_losers". Thanks :)

dohmatob commented:
On Fri, Jan 2, 2015 at 11:17 PM, karinqe notifications@github.com wrote:

> Hi, thanks for the pull request. Are you actually somehow using the bot? I
> thought the robopoker competition got discontinued.
> 
> Yeah, moments after I stumbled on your code, I found out that robopoker got
> discontinued. On my side, I'm working on some formal methods (unpublished
> WIP) for computing nash equilibria and recently I've been looking around
> for a bot into which I could feed the optimal strategy, without having to
> write one from scratch myself. I was shocked by the simplicity of your
> stuff: you just have to sub-class SimpleBot and crush the 'decide' method!
> Clean!
> You're definitely right about the naming :)

As for the patch, could you maybe rename it to something like

> "log_winners_and_losers". Thanks :)
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/karinqe/poker/pull/1#issuecomment-68567892.

## 

DED

dohmatob commented:
Done (the renaming). You should have received a notification.

dohmatob committed:
TYPO: looser -> loser

karinqe commented:
Um, sorry for nit-picking, but it's spelled "loser" not "looser" and it looks a bit weird like this...

dohmatob commented:
You're not nit-picking. I've fixed the typo. Thanks.

dohmatob committed:
patched files to be compliant with pep (closes issue #2)

dohmatob committed:
BF: multiple pep issues
BF: removed Python reserved words from code
CLEANUP: removed some junk code here and there

dohmatob commented:
Hi there,

Do you plan to merge this PR any time soon ?

karinqe commented:
Sorry, I was without internet for a few days.

As for the pep8, I was kinda wondering if I forgot to check my code with it, because I usually do - now I see that the issue is with the RoboPoker SDK code. That is just this code https://github.com/robopoker/robopoker copied over to mine to a `robopoker` dir, because I was in a hurry and didn't have time to fork&fix that repo, plus set up pip installation for it (and add it as a dep to mine), which would be the correct way of doing it. Please don't modify that directory, I only changed the minimum of things to get it working, so I can easily do it the correct way later. If you want to modify that code, please do it in a fork of [robopoker](https://github.com/robopoker/robopoke) and set it up correctly.

Anyway, I'm currently not really developing this bot anymore, so you can just keep the changes in your fork and not really worry about doing pull requests. You should still be able to pull my changes in case I continue developing it (unlikely, since the competition was discontinued).
